### PR TITLE
Fix #233 - ventilation design changes

### LINKED ISSUE #233: Ventilation - Move title outside of cell
Ventilation screens have section headings inside the cell instead out outside it.
This is inconsistent to Daily Rounds and Your Welfare

Note: Recommend doing this task together with #230 if it’s still open

Headings affected
- [ ] **A** Initial Steps - Lung protection
- [ ] **B** Initial Steps - Aims
- [ ] **C** Initial Steps - If deterioritating
- [ ] **D** Adjuncts - If initial stepwise approach fails try

Screenshot (132)

Example
IMG_B96A6315D47D-1

### PR DESCRIPTION
**Pull request by rockgecko-development**
Redesign ventilation page so that the first tab is like the intubation guide, and the second and third tabs are like ICU daily round tabs.

## Screenshots

image
image

## Things to note

`VentilationView` can now render a list of `IntubationContent` and `ICUDailyRoundSteps` items

### CODE REVIEW

File: lib/view/icu_non_intensivist/ventilation/ventilation_view.dart [hunk 1]
```diff
@@ -17,24 +20,39 @@ class VentilationView extends StatelessWidget {
...
-          onPressed: () =>
-              Navigator.of(context).pushNamed(Routes.ventilationInfographic),
-        )
-      ],
-      children: ventilationGuide
-          .map((e) => VentilationGuideContainer(guideSections: e.guideList))
-          .toList(),
-    );
+    final tabs = ventilationGuide.map((e) {
+      if (e is IntubationContent) {
```
Comment (maks) [inline lib/view/icu_non_intensivist/ventilation/ventilation_view.dart:24]: what does the if-else do if both branches return `e.heading` ?

File: lib/view/icu_non_intensivist/ventilation/ventilation_view.dart [hunk 2]
```diff
@@ -17,24 +20,39 @@ class VentilationView extends StatelessWidget {
...
+        ],
+        children: ventilationGuide.map((e) {
+          if (e is IntubationContent) {
+            return IntubationContentViewTemplate(
+              content: e,
+              color: AppColors.blue500,
+            );
+          } else if (e is ICUDailyRoundSteps) {
+            return ICUDailyRoundStepsContainer(
+                steps: e, backgroundColor: AppColors.blue500);
```
Comment (maks) [inline lib/view/icu_non_intensivist/ventilation/ventilation_view.dart:52]: trailing comma pls :slightly_smiling_face:

File: lib/view/icu_non_intensivist/ventilation/ventilation_view.dart [hunk 3]
```diff
@@ -17,24 +20,39 @@ class VentilationView extends StatelessWidget {
...
+          } else {
+            throw Exception('Unknown model ${e.runtimeType}');
```
Comment (maks) [inline lib/view/icu_non_intensivist/ventilation/ventilation_view.dart]: :+1:

**Review by maks — CHANGES_REQUESTED** [review]
great work @rockgecko-development !

File: lib/view/icu_non_intensivist/ventilation/ventilation_view.dart (hunk 1 above)
Comment (rockgecko-development) [reply lib/view/icu_non_intensivist/ventilation/ventilation_view.dart:24]: they're different types. 
Would you prefer `final tabs= ventilationGuide.map((dynamic e) => e.heading as String).toList();`? Might read less clearly, and prone to breaking with any refactor

File: lib/view/icu_non_intensivist/ventilation/ventilation_view.dart (hunk 2 above)
Comment (workerbee22) [reply lib/view/icu_non_intensivist/ventilation/ventilation_view.dart:52]: Yes please to trailing commas.

File: lib/view/icu_non_intensivist/ventilation/ventilation_view.dart (hunk 3 above)
Comment (workerbee22) [reply lib/view/icu_non_intensivist/ventilation/ventilation_view.dart]: Yep we need to start trapping many more exceptions 👍

**Review by workerbee22 — APPROVED** [review]
LGTM ... with Maks comments 👍Nice consolidation.

**Review by ericjiang97 — APPROVED** [review]

File: lib/hard_data.dart [hunk 4]
```diff
@@ -513,88 +512,87 @@ const List<IntubationContent> extubationGuide = [
...
+    ICUDailyRoundStepSection(heading: 'Aims', subsections: [
+      ICUDailyRoundStepSubsection(
         list: [
-          VentilationGuideItemListItem(
-              title: 'ICU consultant review!', icon: '🚨'),
-          VentilationGuideItemListItem(
-              title: 'Proceed to Adjuncts page', icon: '👉'),
+          ICUDailyRoundItem(title: 'SaO₂ 88-92%', icon: '😅'),
+          ICUDailyRoundItem(title: 'PaO₂ 55-70mmHg', icon: '😬'),
+          ICUDailyRoundItem(title: 'Pplat <30', icon: '🙃'),
```
Comment (kimngyn) [inline lib/hard_data.dart]:
```suggestion
          ICUDailyRoundItem(title: 'Pplat < 30', icon: '🙃'),
```

File: lib/hard_data.dart [hunk 5]
```diff
@@ -513,88 +512,87 @@ const List<IntubationContent> extubationGuide = [
...
+          ICUDailyRoundItem(
+              title: 'Ph>7.20',
```
Comment (kimngyn) [inline lib/hard_data.dart]:
```suggestion
              title: 'pH > 7.20',
```

File: lib/view/icu_non_intensivist/ventilation/ventilation_view.dart (hunk 1 above)
Comment (maks) [reply lib/view/icu_non_intensivist/ventilation/ventilation_view.dart:24]: ah right thats tricky but I understand now.

**Review by maks — APPROVED** [review]
:+1:

**Pull request merged**